.lintrunner/grep_linter.py

```diff
@@ -0,0 +1,348 @@
+# From PyTorch:
```

Comment:
Does grep have settings in toml?

Reply:
NOQA, SPACES and TABS. I think what they look for is apparent from the config. LMK if you need more info!